docs/extensions/readme.md

```diff
@@ -1477,6 +1478,27 @@ headers:
 
 What is returned to users is just `key: value`.
 
+## x-ms-internal
+
+When an operation contains this extensions, the operation will be generated but not be exposed to user in generated clients.
```

Comment:
I would have a sentence of the why you would want to do that, for instance "In situation when the method needs to be wrapped by manual code" or something like that.

Reply:
Added.